From: Ileana Dumitrescu <ileanadumitrescu95@gmail.com>
Date: Mon, 02 Oct 2023 17:13:42 +0300
Forwarded: not-needed
Description: Alter test output files for other architectures
 Fixes test check/chk_fhan0 by altering test output files that diff
  against other architectures.

--- a/check/TP00-sol.cas.out-archi386-4
+++ b/check/TP00-sol.cas.out-archi386-4
@@ -33,12 +33,12 @@
 4.0*(x+0.49999999999999788349552193532316633337295399991575235731193338637114148955797)*(x+0.80901699437494742410229341718281905886015458990288143106772431135263023140945+0.58778525229247312916870595463907276859765243764314599107227248075727847416235*I)*(x+0.80901699437494742410229341718281905886015458990288143106772431135263023140945-0.58778525229247312916870595463907276859765243764314599107227248075727847416235*I)*(x-0.30901699437494742410229341718281905886015458990288143106772431135263023140945+0.95105651629515357211643933337938214340569863412575022244730564443015317008520*I)*(x-0.30901699437494742410229341718281905886015458990288143106772431135263023140945-0.95105651629515357211643933337938214340569863412575022244730564443015317008520*I)*(x+0.50000000000000423652873683718280600296014446437031988012842759272211691344594+0.31757869792100789016825214323025983151078592313581708079521071698994632616715e-15*I),
 0,
 (2*x+1)^2*(x^4+x^3+x^2+x+1),
-4.0*(x+0.49999999999999788349552193532316633337295399991575235731193338637114148955797)*(x+0.50000000000000423652873683718280600296014446437031988012842759272211691344594+0.31757869792100789016825214323025983151078592313581708079521071698994632616715e-15*I)*(x^2-0.61803398874989484820458683436563811772030917980576286213544862270526046281890*x+1.0000000000000000000000000000000000000000000000000000000000000000000000000000)*(x^2+1.6180339887498948482045868343656381177203091798057628621354486227052604628189*x+1.0000000000000000000000000000000000000000000000000000000000000000000000000000),
+4.0*(x+0.50000000000000423652873683718280600296014446437031988012842759272211691344594)*(x+0.49999999999999788349552193532316633337295399991575235731193338637114148955797)*(x^2-0.61803398874989484820458683436563811772030917980576286213544862270526046281890*x+1.0000000000000000000000000000000000000000000000000000000000000000000000000000)*(x^2+1.6180339887498948482045868343656381177203091798057628621354486227052604628189*x+1.0000000000000000000000000000000000000000000000000000000000000000000000000000),
 (2*x+1)^2*(x^4+x^3+x^2+x+1),
 (X-1)*(X+1)*(X^2+1)*(X^2+(-sqrt(3))*X+1)*(X^2-X+1)*(X^2+X+1)*(X^2+sqrt(3)*X+1),
-(X-1)*(X+1)*(X-I)*(X+((-I)*sqrt(3)-1)/2)*(X+((-I)*3+3*sqrt(3))/6)*(X+(I*3-3*sqrt(3))/6)*(X+I)*(X+(I*sqrt(3)+1)/2)*(X+((-I)*sqrt(3)+1)/2)*(X+((-I)*3-3*sqrt(3))/6)*(X+(I*sqrt(3)-1)/2)*(X+(I*3+3*sqrt(3))/6),
+(X-1)*(X+1)*(X-I)*(X+((-I)*sqrt(3)-1)/2)*(X+((-I)*3+3*sqrt(3))/6)*(X+((-I)*sqrt(3)+1)/2)*(X+((-I)*3-3*sqrt(3))/6)*(X+(I*3-3*sqrt(3))/6)*(X+(I*sqrt(3)-1)/2)*(X+I)*(X+(I*sqrt(3)+1)/2)*(X+(I*3+3*sqrt(3))/6),
 (X-1)*(X+1)*(X^2+1)*(X^2-X+1)*(X^2+X+1)*(X^4-X^2+1),
-(X-1)*(X+1)*(X-I)*(X+((-I)*sqrt(3)-1)/2)*(X+((-I)*3+3*sqrt(3))/6)*(X+(I*3-3*sqrt(3))/6)*(X+I)*(X+(I*sqrt(3)+1)/2)*(X+((-I)*sqrt(3)+1)/2)*(X+((-I)*3-3*sqrt(3))/6)*(X+(I*sqrt(3)-1)/2)*(X+(I*3+3*sqrt(3))/6),
+(X-1)*(X+1)*(X-I)*(X+((-I)*sqrt(3)-1)/2)*(X+((-I)*3+3*sqrt(3))/6)*(X+((-I)*sqrt(3)+1)/2)*(X+((-I)*3-3*sqrt(3))/6)*(X+(I*3-3*sqrt(3))/6)*(X+(I*sqrt(3)-1)/2)*(X+I)*(X+(I*sqrt(3)+1)/2)*(X+(I*3+3*sqrt(3))/6),
 1+I*sqrt(3),
 exp(2*I*Pi/9),
 0,
--- a/check/TP00-sol.cas.out-archarmel-5
+++ b/check/TP00-sol.cas.out-archarmel-5
@@ -33,12 +33,12 @@
 4.0*(x+0.49999999999999788349552099382283094073616854227304830384829676320412646603489)*(x+0.80901699437494742410229341718281905886015458990288143106772431135263023140945+0.58778525229247312916870595463907276859765243764314599107227248075727847416235*I)*(x+0.80901699437494742410229341718281905886015458990288143106772431135263023140945-0.58778525229247312916870595463907276859765243764314599107227248075727847416235*I)*(x-0.30901699437494742410229341718281905886015458990288143106772431135263023140945+0.95105651629515357211643933337938214340569863412575022244730564443015317008520*I)*(x-0.30901699437494742410229341718281905886015458990288143106772431135263023140945-0.95105651629515357211643933337938214340569863412575022244730564443015317008520*I)*(x+0.50000000000000423652873866744096133134925237086727128317399273251261816211367+0.31757869575771877431763872202300318623834849619459415460408363224777378859822e-15*I),
 0,
 (2*x+1)^2*(x^4+x^3+x^2+x+1),
-4.0*(x+0.49999999999999788349552099382283094073616854227304830384829676320412646603489)*(x+0.50000000000000423652873866744096133134925237086727128317399273251261816211367+0.31757869575771877431763872202300318623834849619459415460408363224777378859822e-15*I)*(x^2-0.61803398874989484820458683436563811772030917980576286213544862270526046281890*x+1.0000000000000000000000000000000000000000000000000000000000000000000000000000)*(x^2+1.6180339887498948482045868343656381177203091798057628621354486227052604628189*x+1.0000000000000000000000000000000000000000000000000000000000000000000000000000),
+4.0*(x+0.50000000000000423652873866744096133134925237086727128317399273251261816211367)*(x+0.49999999999999788349552099382283094073616854227304830384829676320412646603489)*(x^2-0.61803398874989484820458683436563811772030917980576286213544862270526046281890*x+1.0000000000000000000000000000000000000000000000000000000000000000000000000000)*(x^2+1.6180339887498948482045868343656381177203091798057628621354486227052604628189*x+1.0000000000000000000000000000000000000000000000000000000000000000000000000000),
 (2*x+1)^2*(x^4+x^3+x^2+x+1),
 (X-1)*(X+1)*(X^2+1)*(X^2+(-sqrt(3))*X+1)*(X^2-X+1)*(X^2+X+1)*(X^2+sqrt(3)*X+1),
-(X-1)*(X+1)*(X-I)*(X+((-I)*sqrt(3)-1)/2)*(X+((-I)*3+3*sqrt(3))/6)*(X+(I*3-3*sqrt(3))/6)*(X+I)*(X+(I*sqrt(3)+1)/2)*(X+((-I)*sqrt(3)+1)/2)*(X+((-I)*3-3*sqrt(3))/6)*(X+(I*sqrt(3)-1)/2)*(X+(I*3+3*sqrt(3))/6),
+(X-1)*(X+1)*(X-I)*(X+((-I)*sqrt(3)-1)/2)*(X+((-I)*3+3*sqrt(3))/6)*(X+((-I)*sqrt(3)+1)/2)*(X+((-I)*3-3*sqrt(3))/6)*(X+(I*3-3*sqrt(3))/6)*(X+(I*sqrt(3)-1)/2)*(X+I)*(X+(I*sqrt(3)+1)/2)*(X+(I*3+3*sqrt(3))/6),
 (X-1)*(X+1)*(X^2+1)*(X^2-X+1)*(X^2+X+1)*(X^4-X^2+1),
-(X-1)*(X+1)*(X-I)*(X+((-I)*sqrt(3)-1)/2)*(X+((-I)*3+3*sqrt(3))/6)*(X+(I*3-3*sqrt(3))/6)*(X+I)*(X+(I*sqrt(3)+1)/2)*(X+((-I)*sqrt(3)+1)/2)*(X+((-I)*3-3*sqrt(3))/6)*(X+(I*sqrt(3)-1)/2)*(X+(I*3+3*sqrt(3))/6),
+(X-1)*(X+1)*(X-I)*(X+((-I)*sqrt(3)-1)/2)*(X+((-I)*3+3*sqrt(3))/6)*(X+((-I)*sqrt(3)+1)/2)*(X+((-I)*3-3*sqrt(3))/6)*(X+(I*3-3*sqrt(3))/6)*(X+(I*sqrt(3)-1)/2)*(X+I)*(X+(I*sqrt(3)+1)/2)*(X+(I*3+3*sqrt(3))/6),
 1+I*sqrt(3),
 exp(2*I*Pi/9),
 0,
--- a/check/TP00-sol.cas.out-arm64-6
+++ b/check/TP00-sol.cas.out-arm64-6
@@ -33,12 +33,12 @@
 4.0*(x+0.49999999999999788336040210854407999414736036875166481227689775849574875785966)*(x+0.80901699437494742410229341718281905886015458990288143106772431135263023140945+0.58778525229247312916870595463907276859765243764314599107227248075727847416235*I)*(x+0.80901699437494742410229341718281905886015458990288143106772431135263023140945-0.58778525229247312916870595463907276859765243764314599107227248075727847416235*I)*(x-0.30901699437494742410229341718281905886015458990288143106772431135263023140945+0.95105651629515357211643933337938214340569863412575022244730564443015317008520*I)*(x-0.30901699437494742410229341718281905886015458990288143106772431135263023140945-0.95105651629515357211643933337938214340569863412575022244730564443015317008520*I)*(x+0.50000000000000423680523789391990471302495476233592134011221496165941363126275+0.31785500944345246535434955207308258105681155206751838838200313816770312660366e-15*I),
 0,
 (2*x+1)^2*(x^4+x^3+x^2+x+1),
-4.0*(x+0.49999999999999788336040210854407999414736036875166481227689775849574875785966)*(x+0.50000000000000423680523789391990471302495476233592134011221496165941363126275+0.31785500944345246535434955207308258105681155206751838838200313816770312660366e-15*I)*(x^2-0.61803398874989484820458683436563811772030917980576286213544862270526046281890*x+1.0000000000000000000000000000000000000000000000000000000000000000000000000000)*(x^2+1.6180339887498948482045868343656381177203091798057628621354486227052604628189*x+1.0000000000000000000000000000000000000000000000000000000000000000000000000000),
+4.0*(x+0.50000000000000423680523789391990471302495476233592134011221496165941363126275)*(x+0.49999999999999788336040210854407999414736036875166481227689775849574875785966)*(x^2-0.61803398874989484820458683436563811772030917980576286213544862270526046281890*x+1.0000000000000000000000000000000000000000000000000000000000000000000000000000)*(x^2+1.6180339887498948482045868343656381177203091798057628621354486227052604628189*x+1.0000000000000000000000000000000000000000000000000000000000000000000000000000),
 (2*x+1)^2*(x^4+x^3+x^2+x+1),
 (X-1)*(X+1)*(X^2+1)*(X^2+(-sqrt(3))*X+1)*(X^2-X+1)*(X^2+X+1)*(X^2+sqrt(3)*X+1),
-(X-1)*(X+1)*(X-I)*(X+((-I)*sqrt(3)-1)/2)*(X+((-I)*3+3*sqrt(3))/6)*(X+(I*3-3*sqrt(3))/6)*(X+I)*(X+(I*sqrt(3)+1)/2)*(X+((-I)*sqrt(3)+1)/2)*(X+((-I)*3-3*sqrt(3))/6)*(X+(I*sqrt(3)-1)/2)*(X+(I*3+3*sqrt(3))/6),
+(X-1)*(X+1)*(X-I)*(X+((-I)*sqrt(3)-1)/2)*(X+((-I)*3+3*sqrt(3))/6)*(X+((-I)*sqrt(3)+1)/2)*(X+((-I)*3-3*sqrt(3))/6)*(X+(I*3-3*sqrt(3))/6)*(X+(I*sqrt(3)-1)/2)*(X+I)*(X+(I*sqrt(3)+1)/2)*(X+(I*3+3*sqrt(3))/6),
 (X-1)*(X+1)*(X^2+1)*(X^2-X+1)*(X^2+X+1)*(X^4-X^2+1),
-(X-1)*(X+1)*(X-I)*(X+((-I)*sqrt(3)-1)/2)*(X+((-I)*3+3*sqrt(3))/6)*(X+(I*3-3*sqrt(3))/6)*(X+I)*(X+(I*sqrt(3)+1)/2)*(X+((-I)*sqrt(3)+1)/2)*(X+((-I)*3-3*sqrt(3))/6)*(X+(I*sqrt(3)-1)/2)*(X+(I*3+3*sqrt(3))/6),
+(X-1)*(X+1)*(X-I)*(X+((-I)*sqrt(3)-1)/2)*(X+((-I)*3+3*sqrt(3))/6)*(X+((-I)*sqrt(3)+1)/2)*(X+((-I)*3-3*sqrt(3))/6)*(X+(I*3-3*sqrt(3))/6)*(X+(I*sqrt(3)-1)/2)*(X+I)*(X+(I*sqrt(3)+1)/2)*(X+(I*3+3*sqrt(3))/6),
 1+I*sqrt(3),
 exp(2*I*Pi/9),
 0,
